packages/ripple/src/getRippleGlobalNamespace.ts

```diff
@@ -0,0 +1,37 @@
+interface ModuleType {
+  '@leafygreen-ui/ripple': {
+    // @ts-expect-error using HTMLElements to index as it provides a faster lookup when deciding if we should create a ripple effect on a given element
+    registeredRippleElements: { [K: HTMLElement]: {} };
+    setRippleListener: boolean;
+  };
+}
+
+type LGWindow = Window &
+  typeof globalThis & {
+    __LEAFYGREEN_UTILS__: {
+      modules: ModuleType;
+    };
+  };
+
+export function getRippleGlobalNamespace() {
+  if (!window) {
+    return;
+  }
+
+  const rippleModule = {
+    '@leafygreen-ui/ripple': {
+      setRippleListener: false,
+      registeredRippleElements: {},
+    },
+  };
+
+  if ((window as LGWindow).__LEAFYGREEN_UTILS__) {
+    (window as LGWindow).__LEAFYGREEN_UTILS__.modules = rippleModule;
+  }
+
+  (window as LGWindow).__LEAFYGREEN_UTILS__ = { modules: rippleModule };
```

Comment:
I'm confused how this last line won't just always overwrite the previous lines.

Reply:
Yep, fixed this